frontend/src/pages/add-book.jsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import axios from 'axios';
+import { URL_BOOK_SVC, URL_COLLECTION_SVC } from '../configs.js';
+import {
+  Box,
+  Button,
+  ImageListItem,
+  TextField,
+  Typography
+} from '@mui/material';
+
+const AddBook = () => {
+  const [book, setBook] = useState([]);
+  const [price, setPrice] = useState([]);
+  const [note, setNote] = useState([]);
+  const user_id = localStorage.getItem('userId');
+  const { book_id } = useParams();
+  const navigate = useNavigate();
+
+  const handlePriceChange = (e) => {
+    setPrice(e.target.value);
+  }
```

Comment:
this is ambiguous but i think there should be ; at the end of const declarations
i follow this so you might want to do that for consistency

Reply:
hm ye i just noticed that you were doing that too when i coded this. but isn't it a func declation tho haha just saying :0

Reply:
hm also i think i didn't put semicolons for things like these in book-service? (i think we had a convo abt smth like there shouldn't be a ; after } so func declarations don't need one)